radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/forks/ForkBuilder.java

```diff
@@ -72,16 +72,17 @@
  * parameters before the actual fork config object is created.
  */
 public final class ForkBuilder {
```

Comment:
Again a bit unrelated, but what's the advantage of making a Builder immutable?

Reply:
same as for any other immutable structure :P one example: you can share a partially initialized builders

Reply:
I would argue it is a bit of overengineering in this case.